### DISCUSSION TITLE
Error when executing npm run dev

### DISCUSSION THREAD

**Discussion by marktophawk** [vercel/next.js#43851] [Help]
OS :Windows 7 64bit

Nodejs : v13.14.0

After I executed below comman lines:
npx create-next-app@latest nextjs-blog --use-npm --example "https://github.com/vercel/next-learn/tree/master/basics/learn-starter"
cd nextjs-blog
npm run dev 

The console appeared error information:

```
C:\Users\eunice\Downloads\react\nextjs-blog\node_modules\next\dist\cli\next-dev.
js:315
            showAll: args["--show-all"] ?? false,
                                         ^

SyntaxError: Unexpected token '?'
[90m    at Object.compileFunction (vm.js:344:18)[39m
[90m    at wrapSafe (internal/modules/cjs/loader.js:1048:15)[39m
[90m    at Module._compile (internal/modules/cjs/loader.js:1082:27)[39m
[90m    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1138:1
0)[39m
[90m    at Module.load (internal/modules/cjs/loader.js:982:32)[39m
[90m    at Function.Module._load (internal/modules/cjs/loader.js:875:14)[39m
[90m    at Module.require (internal/modules/cjs/loader.js:1022:19)[39m
[90m    at require (internal/modules/cjs/helpers.js:72:18)[39m
    at Object.dev (C:\Users\eunice\Downloads\react\nextjs-blog\node_modules\[4m
next[24m\dist\lib\commands.js:10:30)
    at Object.<anonymous> (C:\Users\eunice\Downloads\react\nextjs-blog\node_modu
les\[4mnext[24m\dist\bin\next:141:28)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ dev: `next dev`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ dev script.
npm ERR! This is probably not a problem with npm. There is likely additional log
ging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\eunice\AppData\Roaming\npm-cache\_logs\2022-12-08T15_45_31
_858Z-debug.log
```

It seems occour at this file:
node_modules\next\dist\cli\next-dev.js

I tried to trace and compared pervious version well executed:
There are many different between them.
Especially here:
        ```
let server = bindings.turbo.startDev({
            ...devServerOptions,
            showAll: args["--show-all"] ?? false,
            root: args["--root"] ?? (turboJson ? _path.default.dirname(turboJson) : packagePath ? _path.default.dirname(packagePath) : undefined),
            serverComponentsExternalPackages: (ref1 = rawNextConfig.experimental) == null ? void 0 : ref1.serverComponentsExternalPackages
        });
```

**Comment by ciruz** [comment]
If you install next.js with this command: ``npx create-next-app@latest nextjs-blog --use-npm --example "https://github.com/vercel/next-learn/tree/master/basics/learn-starter"``, you will install the latest next.js version (``13.x``) which would need at least Node.js in version `14.0.0` (see the docs for `next.js 13` - https://nextjs.org/docs/upgrading#v13-summary ):

> The minimum Node.js version has been bumped from 12.22.0 to 14.0.0, since 12.x has reached end-of-life.

So first update your Node.js to at least ``14.0.0``, and it will probably work right away.

**Reply by GitGabri** [reply]
Hi, I am getting a very similar issue, but the answers here did not fix it for me. I was wondering if you have any insights. Here is the error message in the terminal.

/home/gabsos/mechbook/node_modules/next/dist/cli/next-dev.js:333                                                                              
showAll: args["--show-all"] ?? false,
                                            ^                                                                                                                                                                                                                          SyntaxError: Unexpected token ?                                                                                                       
at Module._compile (internal/modules/cjs/loader.js:723:23)                                                                        
at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)                                                         
 at Module.load (internal/modules/cjs/loader.js:653:32)                                                                            
at tryModuleLoad (internal/modules/cjs/loader.js:593:12)                                                                          
at Function.Module._load (internal/modules/cjs/loader.js:585:3)                                                                   
at Module.require (internal/modules/cjs/loader.js:692:17)                                                                         
at require (internal/modules/cjs/helpers.js:25:18)                                                                                
at Object.dev (/home/gabsos/mechbook/node_modules/next/dist/lib/commands.js:10:30)                                                
at Object.<anonymous> (/home/gabsos/mechbook/node_modules/next/dist/bin/next:141:28)                                              
at Module._compile (internal/modules/cjs/loader.js:778:30)                                                                    
npm ERR! code ELIFECYCLE                                                                                                         
npm ERR! errno 1                                                                                                                 
 npm ERR! mechbook@0.1.0 dev: `next dev`                                                                                           
npm ERR! Exit status 1                                                                                                          
  npm ERR!                                                                                                                          
npm ERR! Failed at the mechbook@0.1.0 dev script.                                                                                 
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.                                                                                                                                                                  npm ERR! A complete log of this run can be found in:                                                                              
npm ERR!     /home/gabsos/.npm/_logs/2023-01-23T13_12_56_972Z-debug.log 

I tried updating everything and this is the latest that I get: + react@18.2.0 + react-dom@18.2.0 + next@13.1.4  
Should I open up another discussion?

Thanks!
Gabriele
